### DISCUSSION TITLE
How to open different pages in  same one window by vb.net CefSharp?

### DISCUSSION THREAD

**Discussion by chelseynelson2008** [cefsharp/CefSharp#4555] [General]
I create control ChromiumWebBrowser1 in form1.and i open link [www.baidu.com](http://www.baidu.com/) page in ChromiumWebBrowser1. now i find many more links in page page.I click these links again, unfortunately These web links all open in a new pop-up window. I don't want that. I think these links still open in the first window [enter image description here](https://i.stack.imgur.com/E7efj.jpg) Imports CefSharp Imports CefSharp.Handler Imports CefSharp.WinForms.
Imports CefSharp Imports CefSharp.Handler Imports CefSharp.WinForms

Public Class Form1

Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

    ChromiumWebBrowser1.Load("https://www.baidu.com")
End Sub

Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load

End Sub

Private Sub Form1_FormClosing(sender As Object, e As FormClosingEventArgs) Handles MyBase.FormClosing

End Sub

Private Sub bt_Click(sender As Object, e As EventArgs) Handles bt.Click
    TextBox1.Text = ChromiumWebBrowser1.Address
End Sub

**Comment by amaitland** [comment]
Already asked at https://stackoverflow.com/questions/76759701/how-to-open-different-pages-in-the-same-window-by-vb-net-cefsharp
